tools/skiplist/create_skipfile.py

```diff
@@ -0,0 +1,54 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""
+Converts Gerrit Review changed files list to CodeChecker skipfile.
```

Comment:
Please follow the structure of our other tools:
- create a readme in the root directory of the tool which describe what this tool does, how you can use it and maybe an example.
- create a codechecker_skiplist directory which will be a module and you move this file also to this directory. So in the CodeChecker we can import and use it.
- I recommend you to rename this file because from the path (tools/skiplist/create_skipfile.py ) for example I can't figure out that this tool is for gerrit integration. So for example the `gerrit_changed_files_to_skipfile` or something similar would be a better name choice.

Reply:
I've moved it under the scripts directory as we discussed this will not be a separate tool yet.
I thought about moving it under the bin directory but in that case the create_commands.py script needs to be further complicated and I'm not sure we want that.